### ISSUE TITLE
SetSurfaceProvider .NET 9

### ISSUE THREAD

**Issue by Light407** [Redth/ZXing.Net.Maui#213]
Anyone else receives this error on MAUI .NET 9? Trying to understand how I can resolve this

System.MissingMethodException: 'Method not found: void AndroidX.Camera.Core.Preview.SetSurfaceProvider(AndroidX.Camera.Core.Preview/ISurfaceProvider)'

**Comment by tritter** [comment]
Yes happens on Android 15, think its because of the deprecated Camera view...

**Comment by tritter** [comment]
Probably fix is equal to: https://github.com/CommunityToolkit/Maui/issues/2377

**Comment by tritter** [comment]
Fixed in https://github.com/Redth/ZXing.Net.Maui/pull/216

**Comment by shobanasuresh** [comment]
We are also receiving this error on MAUI .NET 9

Any idea on when https://github.com/Redth/ZXing.Net.Maui/pull/216 will be released?

**Comment by tritter** [comment]
@MarlonnLima  @ArtefactoDev check my fork and PR: https://github.com/Redth/ZXing.Net.Maui/pull/216
